src/cosima/src/MCActivator.cc

```diff
@@ -423,33 +430,38 @@ bool MCActivator::CalculateEquilibriumRates()
               G4Ions* Nucleus = dynamic_cast<G4Ions*>(Tree[b].back().GetDefinition()); 
               
               bool LevelsOK = true;
-              G4NuclearLevelManager* M = G4NuclearLevelStore::GetInstance()->GetManager(Nucleus->GetAtomicNumber(), Nucleus->GetAtomicMass());
-              if (M->IsValid() == true) {
+              //G4NucLearLevelManager* M = G4NucLevelStore::GetInstance()->GetManager(Nucleus->GetAtomicNumber(), Nucleus->GetAtomicMass());
+              const G4LevelManager* M = G4NuclearLevelData::GetInstance()->GetLevelManager(Nucleus->GetAtomicNumber(), Nucleus->GetAtomicMass());
+	      
+	      //if (M->IsValid() == true) {
+	      bool IsValid = true; // IsValid() Method no longer exist for G4LevelManager, is all level are registered now ? 
+	      if (IsValid == true) {
+
                 //M->PrintAll();
-                const G4NuclearLevel* NuclearLevel = M->NearestLevel(Nucleus->GetExcitationEnergy());
-                cout<<"Nearest level: "<<NuclearLevel->Energy()/keV<<" vs. "<<Nucleus->GetExcitationEnergy()/keV<<endl;
+                const G4NucLevel* NuclearLevel = M->NearestLevel(Nucleus->GetExcitationEnergy());
+                cout<<"Nearest level: "<<M->NearestLevelEnergy(Nucleus->GetExcitationEnergy())/keV<<" vs. "<<Nucleus->GetExcitationEnergy()/keV<<endl;
                 if (NuclearLevel != 0) {
                   // Create new levels...
-                  cout<<"Number of gammas: "<<NuclearLevel->NumberOfGammas()<<endl;
-                  for (int h = 0; h < NuclearLevel->NumberOfGammas(); ++h) {
+                  cout<<"Number of gammas: "<<NuclearLevel->NumberOfTransitions()<<endl;
+                  for (int h = 0; h < int(NuclearLevel->NumberOfTransitions()); ++h) {
                     vector<MCActivatorParticle> ABranch = Tree[b];
-                    cout<<"Gamma energy: "<<NuclearLevel->GammaEnergies()[h]/keV<<endl;
+                    cout<<"Gamma energy: "<<M->LevelEnergy(NuclearLevel->FinalExcitationIndex(h))/keV<<endl;
```

Comment:
This is the part where I am less confident here i.e how to access now to the energies of the transition gammas which was handled before by the class `G4NuclearLevel` which no more exist and was replaced by `G4NucLevel`.
After reading release note patch 10.3 and have a look on the `G4NucLevel` and `G4LevelManager` class, i think that you can access to the gamma energy of the h transition with `M->LevelEnergy(NuclearLevel->FinalExcitationIndex(h))`

I recommend that you check also the class and tell me what you think about it.